content/extending-ui/extending-chat-widget/chat-widget-moments/index.mdx

```diff
@@ -182,17 +208,14 @@ Alternatively you can setup the [MessageBox](/extending-ui/extending-agent-app/a
 
 ---
 
-# Changelog
-
-#### [v0.0.3] - 16.08.2019
+## Changelog
 
-- Add Calendly app example
+### [v1.1.0] - 2019-12-17
 
-#### [v0.0.2] - 07.08.2019
+#### Added
 
-- Update methods description with UMD build usage.
-- Describe MessageBox integration as a place to use Moments.
```

Comment:
Why you have removed those version from the changelog?

Reply:
because I think that the `moments-sdk` should be versioned, not the documentation itself, therefore I've updated that and changed the nesting to include that

Reply:
The documentation itself should version and we are doing this in other docs. You should know when and what has changed in the documentation itself.

Reply:
I agree that we should keep versioning here, but I don't think that's valuable information that we've added the `calendly app example`, or `described MessageBox integration`

This is not done this way in any of other documentations, and is unrelated to the actual version of the `moments-sdk`

It's valuable to keep important things like change of the shape, deprecated or added methods, and so on, but the examples being added or removed is unrelated to the actual package version, and because of that the changelog is now nested in the `Moments SDK`, not the `Moments` themselves

WDYT?

Reply:
ok discussed f2f we should keep the changelog in syn with the changes in the package not the content of the doc itself 👍